web/src/main/webapp/WEB-INF/classes/org/fao/geonet/api/Messages.properties

```diff
@@ -54,8 +54,8 @@ user_password_changed='%s' password was updated.
 user_password_notchanged=A problem occurred trying to change '%s' password. Contact the helpdesk.
 user_password_invalid_changekey='%s' is an invalid change key for '%s'. Change keys are only valid for one day.
 user_registered=User '%s' registered.
-user_with_that_email_found=A user with this email or username already exists.
-user_with_that_username_found=A user with this email or username already exists.
+user_with_that_email_found=A user with this email '%s' already exists.
+user_with_that_username_found=A user with this username '%s' already exists.
```

Comment:
You are reverting this change: https://github.com/geonetwork/core-geonetwork/pull/6958, it should contain the previous message.

Reply:
Reverted back to original and applied to test messages